bip-pbst.md

```diff
@@ -0,0 +1,305 @@
+# BIP-XXXX: Programmable Bitcoin Token Standard (PBTS)
+
+```
+BIP: XXXX
+Layer: Applications
+Title: Programmable Bitcoin Token Standard (PBTS)
+Author: Gobi Shanthan <gobi@torram.xyz>, Lee Raj. <lee@torram.xyz>
+Comments-Summary: No comments yet.
+Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-XXXX
+Status: Draft
+Type: Standards Track
+Created: 2025-02-11
+License: BSD-2-Clause
+Discussions-To: bitcoin-dev@lists.linuxfoundation.org
+Post-History: 2025-02-11
```

Comment:
I don’t seem to have any emails received via the Bitcoin Developer Mailing List that pertain to this proposal.